ai_for_good.qmd

```diff
@@ -1,10 +1,11 @@
 # AI for Good
 
-All of this technological discussion around embedded AI and TinyML we have been talking about so far has the potential to transform humanity's greatest challenges. Initiatives under "AI for Good" promote the development of AI to further the UN Sustainable Development Goals using embedded AI technologies, expanding access to AI education, amongst other things. By aligning AI progress with human values, goals, and ethics, the ultimate goal of ML systems (at any scale) is to be a technology that reflects human principles and aspirations.
+By aligning AI progress with human values, goals, and ethics, the ultimate goal of ML systems (at any scale) is to be a technology that reflects human principles and aspirations. Initiatives under "AI for Good" promote the development of AI to tackle the UN Sustainable Development Goals (SDGs) using embedded AI technologies, expanding access to AI education, amongst other things. While it is now clear that AI will be an instrumental part of progress towards the SDGs, its adoption and impact are limited by the immense power consumption, strong connectivity requirements and high costs of cloud-based deployments. TinyML, applowing ML models to run on low-cost and low-power microcontrollers, can circumvent many of these issues.
 
 > The "AI for Good" movement plays a critical role in cultivating a future where an AI-empowered society is more just, sustainable, and prosperous for all of humanity.
 
 ::: {.callout-note collapse="true"}
+
 ## Learning Objectives
 
 * coming soon.
```

Comment:
I will synthesize this as we make progress on it, so don't worry about it for now.